yaki_admin/src/stores/captainStore.ts

```diff
@@ -15,19 +16,26 @@ export const useCaptainStore = defineStore("captainStore", {
     // in the customer view, when a captain is selected to maybe be deleted
     // at delete icon press, the captainID will be saved
     captainToDelete: 0 as number,
+    currentCustomerId: 0 as number,
   }),
   getters: {
     getCaptainList: (state: State) => state.captainList,
     getCaptainToDelete: (state: State) => state.captainToDelete,
+    getCurrentCustomerId: (state: State) => state.currentCustomerId,
   },
+
   actions: {
     //get the captainId to delete
     setCaptainToDelete(captainId: number) {
       this.captainToDelete = captainId;
     },
+    resetCaptainList() {
```

Comment:
What's the use of reseting the captain list to an empty array ?

Reply:
I deleted it